users/users-model.js

```diff
@@ -0,0 +1,34 @@
+const db = require('../data/dbConfig');
+
+module.exports = {
+  find,
+  findById,
+  add,
+  findBy
+}
+
+function find() {
+  return db('users').select('id', 'username', 'password');
+}
+
+function findBy(filter){
+  return db('users').where(filter);
+}
+
+function findById(id){
+  return db('users')
+    .select('id', 'username', 'password')
```

Comment:
Since you have this same pattern where you need users, but without their passwords, in both `find` and `findById`, you could make a helper function for them both (or, in this case, `find` could be used as the helper since it contains all of the similar code):
```javascript
function findById(id) {
  return find()
    .where({ id })
    .first();
}
```